unity-client/Assets/Scripts/MainScripts/DCL/Controllers/Scene/SceneLifecycleHandler.cs

```diff
@@ -0,0 +1,147 @@
+﻿using System.Collections.Generic;
+using DCL.Components;
+using DCL.Models;
+using UnityEngine;
+
+namespace DCL.Controllers
+{
+    public class SceneLifecycleHandler
```

Comment:
As we usually are doing with all the new classes, it would be fine to add a basic XML documentation about the responsibilities of this class :)

Reply:
Done